cypress/integration/video-player.test.js

```diff
@@ -63,14 +66,14 @@ context("Test Video Player interactive", () => {
       });
       phoneListen("interactiveState");
       cy.getIframeBody().find(".vjs-big-play-button").click();
+      cy.getIframeBody().find(".vjs-text-track-cue").should("include.text", "This is a drake");
+      cy.getIframeBody().find(".vjs-current-time-display").should("include.text", "0:01");
+      cy.getIframeBody().find(".vjs-poster.vjs-hidden").should("exist");
+
       getAndClearLastPhoneMessage((state) => {
         expect(state.lastViewedTimestamp).greaterThan(1.0);
-
-        cy.getIframeBody().find(".vjs-text-track-cue").should("include.text", "This is a drake");
-        cy.getIframeBody().find(".vjs-current-time-display").should("include.text", "0:01");
-        cy.getIframeBody().find(".vjs-poster.vjs-hidden").should("exist");
-
-      });
+        // must wait for video to finish apparently
+      }, 15000);
```

Comment:
If I remember video implementation correctly, it should update progress every second. So, theoretically, it'd be enough to wait 1-2 seconds max. I guess you tried smaller values before? When I was testing that in practice in LARA, I wasn't sure if it actually works as designed.